src/components/card/Card.test.js

```diff
@@ -0,0 +1,9 @@
+import Card from './Card'
```

Comment:
**praise:** for writing a test.

**nitpick:** the features that the card provides are not tested.

Reply:
@coderbyheart I actually need to discuss testing with someone. I wanted to write test/s, but running `yarn test` doesn't seem to work. Any recommended solution?